src/System.Runtime.Intrinsics.X86/ref/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netstandard2.0;
```

Comment:
I believe this should just be `netstandard`

Reply:
Thank you, will fix.